src/test/spec/gridfs.rs

```diff
@@ -332,3 +331,49 @@ async fn assert_no_chunks_written(bucket: &GridFsBucket, id: &Bson) {
         .unwrap()
         .is_none());
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn test_gridfs_bucket_find_one() {
+    let data = &[1, 2, 3, 4];
+    let client = TestClient::new().await;
+
+    let options = GridFsBucketOptions::default();
+    let bucket = client.database("gridfs_find_one").gridfs_bucket(options);
+
+    let filename = String::from("somefile");
+    let mut upload_stream = bucket.open_upload_stream(&filename, None);
+    upload_stream.write_all(data).await.unwrap();
+    upload_stream.close().await.unwrap();
+
+    let found = bucket
+        .find_one(doc! { "_id": upload_stream.id() }, None)
+        .await
+        .unwrap()
+        .unwrap();
+
+    assert_eq!(&found.id, upload_stream.id());
+    assert_eq!(found.length, 4);
+    assert_eq!(found.filename, Some(filename));
+}
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn test_gridfs_find_one_options_from() {
```

Comment:
This can be a regular test with the `#[test]` annotation since it doesn't contain any async code, i.e.
```
#[test]
fn test_gridfs_find_one_options_from() {
    ...
}
```